lib/irb/init.rb

```diff
@@ -146,9 +146,12 @@ def IRB.parse_opts(argv: ::ARGV)
         opt = $1 || argv.shift
         @CONF[:LOAD_MODULES].push opt if opt
       when /^-I(.+)?/
-        opt = $1 || argv.shift
-        IRB.fail(LoadPathDoesNotExist) if opt.nil? || !File.directory?(opt)
-        load_path.concat(opt.split(File::PATH_SEPARATOR))
+        opt = ($1 || argv.shift).to_s
+        IRB.fail(LoadPathArgumentMissing) if opt.strip.empty?
```

Comment:
Is `strip`ping right?
space-only names can be valid directory names (I agree it is insane, though).